CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [68.3.0](https://github.com/Sage/carbon/compare/v68.2.0...v68.3.0) (2021-03-12)
```

Comment:
Why does it look like you're changing the link component?

Reply:
Just pulled changes from master - not sure why it's showing as change. It's the same as on master.

Reply:
I understand it's in master, but, I'd not expect your commit to include changes that are already in master.
Has a rebase/squash gone wrong?

If you reset HEAD~1 and remove the files you don't need, does that ensure master is still correct and your PR has the correct changes?

Reply:
It worked as it should - thanks!